packages/ui/src/components/va-carousel/VaCarousel.stories.ts

```diff
@@ -1,12 +1,224 @@
+  template: `
+    [true]
+    <VaCarousel :items="items" :arrows="false" v-model="current"/>
+    [false]
+    <VaCarousel :items="items" :arrows="false" :indicators="false" v-model="current"/>
+  `,
+})
+
+Indicators.play = async ({ canvasElement, step }) => {
+  const canvas = within(canvasElement)
+  const slides = document.querySelectorAll('div.va-carousel__slide')
+  const NavigatesOnPageClick = canvas.getByRole('button', { name: 'go slide 2' }) as HTMLElement
+
+  await step('Should head to slide 2 by clicking on indicator', async () => {
+    await userEvent.click(NavigatesOnPageClick)
+    expect(slides[1].getAttribute('aria-current')).toEqual('true')
+  })
+}
+
+export const IndicatorsTrigger = () => ({
+  components: { VaCarousel },
+  data: () => ({ items: getItems() }),
+  template: `
+    [hover]
+    <VaCarousel :items="items" :arrows="false" indicator-trigger="hover"/>
+    [default]
+    <VaCarousel :items="items" :arrows="false" />
+  `,
+})
+
+IndicatorsTrigger.play = async ({ canvasElement, step }) => {
+  const canvas = within(canvasElement)
+  const slides = document.querySelectorAll('div.va-carousel__slide')
+  const NavigatesOnPageClick = canvas.getAllByRole('button', { name: 'go slide 2' }) as HTMLElement[]
+
+  await step('Should head to slide 2 by hover on indicator', async () => {
+    await userEvent.hover(NavigatesOnPageClick[0])
+    await userEvent.unhover(NavigatesOnPageClick[0])
+    expect(slides[1].getAttribute('aria-current')).toEqual('true')
+  })
+
+  await step('Should remain on slide 1 by hover on indicator', async () => {
+    await userEvent.hover(NavigatesOnPageClick[1])
+    await userEvent.unhover(NavigatesOnPageClick[1])
+    expect(slides[3].getAttribute('aria-current')).toEqual('true')
+  })
 }
 
-export const Default = defineComponent({
+export const Infinite = () => ({
   components: { VaCarousel },
-  template: '<VaCarousel/>',
+  data: () => ({ items: getItems() }),
+  template: `
+    [infinite]
+    <VaCarousel :items="items" infinite />
+    [default]
+    <VaCarousel :items="items" />
+  `,
+})
+
+Infinite.play = async ({ canvasElement, step }) => {
+  const canvas = within(canvasElement)
+  const slides = document.querySelectorAll('div.va-carousel__slide')
+  const NavigatesOnArrowClick = canvas.getAllByRole('button', { name: 'go previous slide' }) as HTMLElement[]
+
+  await step('Should head to slide 3 from slide 1', async () => {
+    await userEvent.click(NavigatesOnArrowClick[0])
+    expect(slides[2].getAttribute('aria-current')).toEqual('true')
+  })
+
+  await step('Should have left arrow disabled', async () => {
+    expect(NavigatesOnArrowClick[1]).toBeUndefined()
+  })
+}
+
+export const Stateful = () => ({
+  components: { VaCarousel },
+  data: () => ({ items: getItems() }),
+  template: `
+    [true] - should slide
+    <VaCarousel :items="items" />
+    [false] - should not slide
+    <VaCarousel :items="items" :stateful="false"/>
```

Comment:
<img width="973" alt="image" src="https://github.com/epicmaxco/vuestic-ui/assets/5394573/a1ae0df1-1966-4055-b636-a7ff5d6444b2">

That seems broken to me. Even if you don't pass value - there should be a slide selected.
In this case - it should default to 1st slide.